src/trade-params/FullTradeParams.js

```diff
@@ -396,28 +396,46 @@ export default class FullTradeParams extends Component {
             <div className="trade-params" disabled={disabled}>
                 <DropDown
                     shown={trade.showAssetPicker}
-                    onClose={() => this.updateHelper('showAssetPicker', false, false)}
+                    onClose={(e) => {
+                            e.preventDefault();
```

Comment:
We should rework this in the way I did it for Trade Type, not in that direction. Have a look.